fractions/microprofile/microprofile-fault-tolerance/README.adoc

```diff
@@ -1,8 +1,7 @@
+[[_microprofile_fault_tolerance]]
+= MicroProfile Fault Tolerance
 
-= Microprofile Fault Tolerance Fraction
-:icons: font
-
-This fraction implements the https://github.com/eclipse/microprofile-fault-tolerance[Eclipse Microprofile Fault tolerance API^].
+This fraction implements the https://github.com/eclipse/microprofile-fault-tolerance[Eclipse MicroProfile Fault Tolerance API^].
```

Comment:
Well, this PR was originally about MP rest client... Anyway, I don't think it's redundant - it does not read well to start a sentence without subject.

Reply:
Okay, let's leave it as it for this PR.

It is a consensus in the documentation community that starting a section by _This section, this chapter, this $WHATEVER_ is superfluous unless it really isn't clear from the context. I believe that in this case, it doesn't even have to be a sentence because _An implementation of the Eclipse MicroProfile Fault Tolerance API_ perfectly suffices.